tests/render/renderer/test_notebook_renderer.py

```diff
@@ -377,104 +377,47 @@ def test_render_without_batch_kwargs_uses_batch_kwargs_in_citations(
     obs = NotebookRenderer().render(critical_suite_with_citations)
     assert isinstance(obs, dict)
     expected = {
-        "nbformat": 4,
-        "nbformat_minor": 4,
-        "metadata": {},
-        "cells": [
-            {
-                "cell_type": "markdown",
-                "source": "# Edit Your Expectation Suite\nUse this notebook to recreate and modify your expectation suite:\n\n**Expectation Suite Name**: `critical`\n\nWe'd love it if you **reach out to us on** the [**Great Expectations Slack Channel**](https://greatexpectations.io/slack)",
-                "metadata": {},
-            },
-            {
-                "cell_type": "code",
-                "metadata": {},
-                "execution_count": None,
-                "source": """\
-from datetime import datetime
-import great_expectations as ge
-import great_expectations.jupyter_ux
-from great_expectations.data_context.types.resource_identifiers import ValidationResultIdentifier
-
-context = ge.data_context.DataContext()
+        'nbformat': 4,
+        'nbformat_minor': 4,
+        'metadata': {},
+        'cells': [
+            {'cell_type': 'markdown',
+             'source': "# Edit Your Expectation Suite\nUse this notebook to recreate and modify your expectation suite:\n\n**Expectation Suite Name**: `critical`\n\nWe'd love it if you **reach out to us on** the [**Great Expectations Slack Channel**](https://greatexpectations.io/slack)",
```

Comment:
The diffs on this file are pretty hard to parse - can we talk through them tomorrow?